test/new-e2e/tests/apm/vm_test.go

```diff
@@ -35,6 +35,9 @@ func vmSuiteOpts(tr transport, opts ...awshost.ProvisionerOption) []e2e.SuiteOpt
 
 // TestVMFakeintakeSuiteUDS runs basic Trace Agent tests over the UDS transport
 func TestVMFakeintakeSuiteUDS(t *testing.T) {
+	// FIXME: fix flakiness reaching the agent then enable the test
+	t.Skip("Skipping flaky test")
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You could create a JIRA card or use the incident-25117 as a reference, it helps rolling back the skip once addressed

Reply:
Good idea, thanks!